frontend/keysets/simple-urdu-keyset/simple-urdu-keys-layout.ts

```diff
@@ -0,0 +1,31 @@
+export const unshiftedKeysLayout = [
+  ["`", "۱", "۲", "۳", "۴", "۵", "٦", "۷", "۸", "۹", "۰", "-", "=", "Delete"],
+  ["Tab", "ق", "و", "ع", "ر", "ت", "ے", "ء", "ی", "ہ", "پ", "[", "]", "\\"],
+  ["Caps", "ا", "س", "د", "ف", "گ", "ح", "ج", "ک", "ل", "؛", "'", "Enter"],
+  ["Shift", "ز", "ش", "چ", "ط", "ب", "ن", "م", "،", "۔", "/", "Shift"],
```

Comment:
```suggestion
  ["`", "۱", "۲", "۳", "۴", "۵", "۶", "۷", "۸", "۹", "۰", "ّ", "-", "=", "Delete"],
  ["Tab", "ق", "و", "ع", "ر", "ت", "ے", "ء", "ی", "ہ", "پ", "ُ", "[", "]", "\\"],
  ["Caps", "ا", "س", "د", "ف", "گ", "ح", "ج", "ک", "ل", "َ", "؛", "'", "Enter"],
  ["Shift", "ز", "ش", "چ", "ط", "ب", "ن", "م", "ِ", "،", "۔", "/", "Shift"],
```

- this 6 is not Urdu. `۶` is correct.
- add new keys for diacritics (right of alphabet, left of symbols).